app/generator/content/schedule/part2/get-recovery-description.js

```diff
@@ -0,0 +1,17 @@
+const getRecoveryDescription = (schedule) => {

```

Comment:
Ticket only concerns 0 reductions.

Can you split out the recovery functionality into a separate PR

Reply:
Yes, the ticket only concern 0 reductions. Treating zero reduction involves breaking standard reduction scenario into three possible scenarios (standard-reduction, reduction-to-zero and recovery) , hence the tickets take care of the three possible scenarios, but only implement changes to reduction-to-zero format. There is a ticket for implementing recovery format and it is coming as another PR.